src/MembershipManager.php

```diff
@@ -189,6 +189,9 @@ public function getGroupIds(EntityInterface $entity, $group_type_id = NULL, $gro
       $query = $this->entityTypeManager
         ->getStorage($target_type)
         ->getQuery()
+        // When rebuilding node_access grants then checking node_access is
```

Comment:
The current user matters less -- when rebuilding node access and your groups are nodes then you are trying to retrieve node access records which do not exist because node_access_rebuild have erased the table in the first place...